widget/contrib/gpmdp.lua

```diff
@@ -8,79 +8,96 @@
 
 local helpers             = require("lain.helpers")
 local json                = require("lain.util.dkjson")
+local watch               = require("awful.widget").watch
 local focused             = require("awful.screen").focused
-local pread               = require("awful.util").pread
 local naughty             = require("naughty")
-local wibox               = require("wibox")
 local next, getenv, table = next, os.getenv, table
 
 -- Google Play Music Desktop infos
 -- lain.widget.contrib.gpmdp
 -- requires: curl
 
 local function factory(args)
-    local gpmdp         = { widget = wibox.widget.textbox() }
+    local gpmdp         = {}
     local args          = args or {}
     local timeout       = args.timeout or 2
     local notify        = args.notify or "off"
+    local hover         = args.hover or "off"
     local followtag     = args.followtag or false
     local file_location = args.file_location or
                           getenv("HOME") .. "/.config/Google Play Music Desktop Player/json_store/playback.json"
     local settings      = args.settings or function() end
 
     gpmdp_notification_preset = {
```

Comment:
Remove it and assume default values, or maybe turn it into `gpmdp.notification_preset`.